pkg/apis/apps/validation/validation_test.go

```diff
@@ -127,17 +127,56 @@ func TestValidateStatefulSet(t *testing.T) {
 					}()},
 			},
 		},
+		{
```

Comment:
Aren't these 3 unit tests the same?

Note that validation doesn't use the feature gate, so no need to simulate that. But you have to test nil and non nil policy.

Reply:
Not sure what happened to those tests, thanks.

Done.